src/libcore/ops/try.rs

```diff
@@ -49,3 +49,21 @@ pub trait Try {
     #[unstable(feature = "try_trait", issue = "42327")]
     fn from_ok(v: Self::Ok) -> Self;
 }
+
+#[unstable(feature = "try_trait", issue = "42327")]
```

Comment:
Note that trait impls are insta-stable, so this can't land as-is.

Reply:
I've implemented this properly now.